web/apps/labelstudio/src/components/Form/Elements/Toggle/Toggle.jsx

```diff
@@ -51,23 +52,16 @@ const Toggle = forwardRef(
       >
         {(ref) => (
           <div className={classList.join(" ")}>
-            <input
-              ref={ref}
+            <UiToggle ref={ref}
```

Comment:
note for reviewers: since Toggle uses `FormField` which has business logic - ui `Toggle` should be subbed in here - until `FormField` unification in `core` happens